packages/jest-worker/src/index.ts

```diff
@@ -105,6 +111,20 @@ export default class JestWorker {
     this._bindExposedWorkerMethods(workerPath, this._options);
   }
 
+  private setUpInspector() {
+    // Open V8 Inspector
+    inspector.open();
+
+    const inspectorUrl = inspector.url();
+    let session;
+    if (inspectorUrl !== undefined) {
+      session = new inspector.Session();
+      session.connect();
+      session.post('Debugger.enable');
```

Comment:
this is async

```suggestion
      await util.promisfy(session.post)('Debugger.enable');
```

Reply:
TS is complaining about the return type of `promisify`:   
`Argument of type '"Debugger.enable"' is not assignable to parameter of type '"NodeWorker.disable"'.`

It returns something with this type:
```js
// util.promisify(session.post)('Debugger.enable')
function util.promisify<"NodeWorker.disable", unknown>(fn: (arg1: "NodeWorker.disable", callback: (err: Error | null, result: unknown) => void) => void): (arg1: "NodeWorker.disable") => Promise<unknown> (+13 overloads)
```

Also, what is the best way to handle the asynchronous assignment of the inspector inside `constructor` ?

Reply:
Ah right, I had issues with promisify myself https://github.com/facebook/jest/pull/8596#discussion_r296476038

Manual `new Promise` should work 👍

Regarding best way of async init is probably a setup function of some sort... Unless we can intercept the user's function calls as they are all async?

Reply:
Or exposing a factory function instead of the class directly. Can do that in Jest 27 even if this PR doesn't land in time for it